forc-pkg/src/pkg.rs

```diff
@@ -2055,7 +2077,47 @@ pub fn build(
     profile: &BuildProfile,
     outputs: &HashSet<NodeIx>,
     const_inject_map: &ConstInjectionMap,
-) -> anyhow::Result<Vec<(NodeIx, BuiltPackage)>> {
+) -> anyhow::Result<Vec<(NodeIx, BuildResult)>> {
+    #[allow(clippy::too_many_arguments)]
+    fn compile_util(
```

Comment:
Can we call this `compile_pkg` instead? I think it's good to try to avoid "util" in favour of something more descriptive when we can (I know we have a bunch of util modules all over the place, but this is probably something we should aim to improve).